package.json

```diff
@@ -15,6 +15,8 @@
     "deploy": "./deploy.sh"
   },
   "dependencies": {
+    "totp-generator": "0.0.14",
```

Comment:
What is this dependency?

Reply:
It's the library that we use to generate TOTP codes, since we merged everything into 1 Lambda, the original Lambda has been removed. This dependency is the same one as before, but I've put it into the parent package.json since we follow this standard in check hmrc.

Reply:
I just checked, I think you're right! It must of been a left over because we switched libraries early on and we must have not removed the dependency.